.github/stock exchange1&2/script.js

```diff
@@ -0,0 +1,65 @@
+
+async function getData(query) {
+    try {
+        const endpoint = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=${query}&limit=10&exchange=NASDAQ`;
```

Comment:
Instead of wrting the api URL inside the string in every http request, you should use baseUrl variable